Modules/_ssl.c

```diff
@@ -4712,7 +4712,6 @@ _ssl_MemoryBIO_read_impl(PySSLMemoryBIO *self, int len)
     nbytes = BIO_read(self->bio, PyBytes_AS_STRING(result), len);
     /* There should never be any short reads but check anyway. */
     if ((nbytes < len) && (_PyBytes_Resize(&result, len) < 0)) {
-        Py_DECREF(result);
```

Comment:
Let's rewrite the block to conform to the style of other ``_PyBytes_Resize`` calls. There is no need to check the return value of ``_PyBytes_Resize``, because the function returns ``result`` any way.

```
if (nbytes < len) {
    _PyBytes_Resize(&result, len);
}
```